### DISCUSSION TITLE
Github Page doesn't change after commit

### DISCUSSION THREAD

**Discussion by YaoxiY** [community/community#54090] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I change the code in my Github Page, but when I open the website, it still shows the old version of webpage. What should I do?

**Comment by airtower-luna** [comment, marked as answer]
Check the Actions tab of your repository, you should see logs of the Pages jobs there. Check if there were any errors, and if yes fix them. If the jobs passed and your site still isn't updated after a while (maybe half an hour, try clearing your browser cache, too) it might be an error on the GitHub side, you can try again (by making another commit), and if that doesn't help [report the bug to support](https://support.github.com/contact/bug-report). 

Saying more isn't possible without seeing your repository.

**Reply by YaoxiY** [reply]
https://github.com/YaoxiY/myportfolio
Hi, thanks for your answer, this is the repository I talked about. Github page doesn't show any change over 24h. I checked that there's no error. What else problem it can be?

**Reply by airtower-luna** [reply]
It looks like your site was deployed a few minutes ago: https://github.com/YaoxiY/myportfolio/actions/runs/4846364743

No idea why *now* when the latest commit happened hours ago. Maybe some hiccup in the system? :sweat_smile:

**Reply by YaoxiY** [reply]
it works now! Did you make some changes? Thanks!

**Reply by airtower-luna** [reply]
I didn't do anything. I can't make changes to your repository and I can't look into GitHub systems, I'm just another user. I just observed. :wink:

**Reply by YaoxiY** [reply]
ahah, i see, now I set it as private repository, and my Github page turn out to be 404 (new problem), when I set it back to Public, it still 404, what should I do?

**Reply by airtower-luna** [reply]
It seems to be active again now, so it probably just took a while again.

AFAIK using Pages from a private repo is a paid feature.

**Reply by YaoxiY** [reply]
oh, private repo is paid feature, so when I make it private, it will be invalid, right?

**Reply by airtower-luna** [reply]
*Having* a private repository isn't a paid feature, but *setting up a Pages site from a private repository* is (see the green box right at the top [here](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages)). So that's why your site disappears when you make the repository private.
